2.4/Dockerfile.fedora

```diff
@@ -0,0 +1,78 @@
+FROM registry.fedoraproject.org/f26/s2i-base:latest
```

Comment:
Just wondering, why not f27?

Reply:
There are some difficulties in Fedora infra with releasing F27 variant:

```
$ docker pull registry.fedoraproject.org/f27/s2i-base:latest
Trying to pull repository registry.fedoraproject.org/f27/s2i-base ...
manifest for registry.fedoraproject.org/f27/s2i-base:latest not found
```

https://pagure.io/releng/issue/7217